src/txmempool.h

```diff
@@ -585,6 +586,25 @@ class CTxMemPool
      */
     std::set<uint256> m_unbroadcast_txids GUARDED_BY(cs);
 
+
+    /**
+     * Helper function to calculate all in-mempool ancestors of staged_ancestors and apply
+     * ancestor/descendant limits. Inclusive of entry_size and entry_count.
+     * param@[in]   entry_size          Virtual size to include in the limits.
+     * param@[in]   entry_count         How many entries to include in the limits.
+     * param@[in]   staged_ancestors    Should contain entries in the mempool.
+     * param@[out]  setAncestors        Will be populated with all mempool ancestors.
+     */
+    bool calculateAncestorsAndCheckLimits(size_t entry_size,
+                                          size_t entry_count,
+                                          setEntries& setAncestors,
```

Comment:
The `&` jumps from right to left between commits. Perhaps just have this as `setEntries& setAncestors` in commit _MOVEONLY: add helper function for calculating ancestors and checking limits_

Reply:
Done